### PR TITLE
Docker container version

### PR DESCRIPTION
**Pull request by Max-TheCat**
Containerizes spliit2 using docker. You can now create an image and run the app in a cluster (db + app). Instructions on how to run it in a container were added to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +35,16 @@ The project is open to contributions. Feel free to open an issue or even a pull-
...
 2. `npm install`
-3. Start a PostgreSQL server. You can run `./start-local-db.sh` if you don’t have a server already.
+3. Start a PostgreSQL server. You can run `./scripts/start-local-db.sh` if you don’t have a server already.
 4. Copy the file `.env.example` as `.env`
 5. `npm run dev`
 
+## Run in a container
+1. Run `sudo npm run build-image` to build the docker image from the Dockerfile
+2. Copy the file `container.env.example` as `container.env`
+3. Run `sudo npm run start-container` to start the postgres and the spliit2 containers
```
Comment (scastiel) [inline README.md:45]:
```suggestion
1. Run `npm run build-image` to build the docker image from the Dockerfile
2. Copy the file `container.env.example` as `container.env`
3. Run `npm run start-container` to start the postgres and the spliit2 containers
```

I’m not sure that `sudo` is required, I was able to run the container version without it. Anything I’m missing?

**Review by scastiel — COMMENTED** [review]
Amazing, thanks @Max-TheCat!

File: README.md (hunk 1 above)
Comment (Max-TheCat) [reply README.md:45]: I'm not sure why but I needed it when running under npm but, it makes sense not to have that in the README.

**Comment by ChristopherJohnston** [comment]
Brilliant!

**Comment by ChristopherJohnston** [comment]
Could this also be added to docker hub? This will aid local (and potentially homeassistant) deployments

**Comment by Max-TheCat** [comment]
@ChristopherJohnston totally, I created an issue https://github.com/spliit-app/spliit/issues/60

**Pull request merged**